src/components/Model3DViewer/Model3DViewer.tsx

```diff
@@ -0,0 +1,270 @@
+import { Cognite3DModel, Cognite3DViewer, THREE } from '@cognite/3d-viewer';
+import * as sdk from '@cognite/sdk';
+import React from 'react';
+import { CacheObject, Callback, MouseScreenPosition } from '../../interfaces';
+import {
+  createViewer as originalCreateViewer,
+  parseBoundingBox,
+  setCameraPosition,
+  ViewerEventTypes,
+} from '../../utils';
+
+let createViewer = originalCreateViewer;
+
+type ClickHandler = (position: MouseScreenPosition) => void;
+
+export interface Model3DViewerProps {
+  projectName: string;
```

Comment:
I suggest we remove this and send in an empty string. This was only there for compatibility reasons and is not used. No reason to expose in the wrapper.

Reply:
Got it

Reply:
But i need it here: